.github/workflows/weekly.yml

```
@@ -36,6 +36,10 @@ jobs:
           flink: 1.18-SNAPSHOT,
           jdk: '8, 11, 17',
           branch: main
+        }, {
+          flink: 1.19-SNAPSHOT,
+          jdk: '8, 11, 17',
```

```suggestion
          jdk: '8, 11, 17, 21',
```

for 1.19 jdk21 is supported as well